src/sparseml/exporters/transforms/conv_to_qlinearconv.py

```diff
@@ -0,0 +1,149 @@
+        (input_dequant,) = match.parents[0]
+        weight_init, weight_quant, weight_dequant = match.parents[1]
+        (output_quant,) = match.children[0]
+
+        weight_quantize_params = get_quantization_params(
+            model, weight_quant, include_target=True
+        )
+        # sanity check - matching should ensure this
+        assert weight_quantize_params.target is not None
+
+        # quantize weight
+        quantized_weight = quantize_array(
+            weight_quantize_params.target,
+            weight_quantize_params.scale,
+            weight_quantize_params.zero_point,
+            weight_quantize_params.zero_point.dtype,
+        )
+        quantized_weight_name = "{}.weight_quantized".format(conv_node.name)
+        quantized_weight_initializer = numpy_helper.from_array(
+            quantized_weight, name=quantized_weight_name
+        )
+        model.graph.initializer.append(quantized_weight_initializer)
+
+        # get qconv inputs and outputs
+        qconv_input = input_dequant.input[0]
+        qconv_inputs = [
+            qconv_input,  # x
+            input_dequant.input[1],  # x_scale
+            input_dequant.input[2],  # x_zero_point
+            quantized_weight_name,  # w
+            weight_quant.input[1],  # w_scale
+            weight_quant.input[2],  # w_zero_point
+            output_quant.input[1],  # y_scale
+            output_quant.input[2],  # y_zero_point
+        ]
+
+        if len(conv_node.input) > 2:
+            bias = get_init_by_name(model, conv_node.input[2])
+            if bias is not None:
+                # quantize bias and add it to the qconv inputs
+                bias = numpy_helper.to_array(bias)
+                input_quantize_params = get_quantization_params(
+                    model, input_dequant, include_target=False
+                )
+                bias_scale = input_quantize_params.scale * weight_quantize_params.scale
+                quantized_bias = quantize_array(bias, bias_scale, 0, numpy.int32)
+                quantized_bias_name = "{}.bias_quantized".format(conv_node.name)
+                quantized_bias_initializer = numpy_helper.from_array(
+                    quantized_bias, name=quantized_bias_name
+                )
+                model.graph.initializer.append(quantized_bias_initializer)
+                qconv_inputs.append(quantized_bias_name)
+
+        qconv_output = output_quant.output[0]
+        qconv_name = "{}_quant".format(conv_node.name)
+        qconv_kwargs = {}
+        for attribute in conv_node.attribute:
+            qconv_kwargs.update(attribute_to_kwarg(attribute))
+
+        # create qconv node and add it to graph
+        qconv_node = helper.make_node(
+            "QLinearConv", qconv_inputs, [qconv_output], qconv_name, **qconv_kwargs
+        )
+        model.graph.node.append(qconv_node)
+
+        # delete original conv and folded quantization ops
+        remove_node_and_params_from_graph(model, conv_node)
```

Comment:
per Ben: let's do the removal of the `match.nodes` in the `transformation` logic (outside of this function)